src/stream.py

```diff
@@ -0,0 +1,64 @@
+import netCDF4
+import numpy as np
+import os
+import xarray as xr
+
+PATH_NC = '../data/cmems_20200916_3.nc'
+
+
+def load_nc(path_nc: str=PATH_NC):
+    """
+    Load the .nc file as a xarray.Dataset
+    """
+    assert os.path.isfile(path_nc)
+    nc = xr.open_dataset(path_nc)
+    return nc
+
+
+def get_stream_velocity(nc, lat: float,  lon: float, date: str, coord_range: float=10):
+    """
+    Get stream velocities and their corresponding latitudes and longitudes
+    
+    Params
+    ------
+    nc : xarray.Dataset
+    lat : float
+        Boat position (latitude)
+    lon : float
+        Boat position (longitude)
+    date : str
+        Date in format YYYY-MM-DD
+    coord_range : float, default=10
+        Range for longitude and latitude
+    
+    Returns
+    -------
+    stream_velocities : np.array
+        An (M, N, K)-dimensional array containing the K-dim
+        values of the magnitude. Each MxN matrix
+        element represents the value for the k_th dimension of
+        that magnitude in the m_th, n_th latitude, longitude point
+        on the discrete space.
+    stream_velocities_latitudes : np.array
+        An (M)-dimensional vector with the latitude values corresponding
+        to the discrete points of the grid_array.
+    stream_velocities_longitudes : np.array
+        An (N)-dimensional vector with the longitude values corresponding
+        to the discrete points of the grid_array.
```

Comment:
Dado que es una función exclusiva para las corrientes, creo que se puede explicitar la dimensionalidad de salida y los nombres concretos, ya que esa información no va a cambiar y da la impresión de que puede variar.

```suggestion
        An (M, N, 2)-dimensional array containing the 2-dim
        values of the stream velocities. Each MxN matrix represents the
        component of the corresponding velocity direction for each discrete lat, long point.
    stream_velocities_latitudes : np.array
        An (M)-dimensional vector with the latitude values corresponding
        to the discrete points of the stream_velocities.
    stream_velocities_longitudes : np.array
        An (N)-dimensional vector with the longitude values corresponding
        to the discrete points of the stream_velocities.
```